apis/core.oam.dev/v1alpha1/policy_types.go

```diff
@@ -77,3 +79,11 @@ func (in SharedResourcePolicySpec) FindStrategy(manifest *unstructured.Unstructu
 	}
 	return false
 }
+
+// ReplicationPolicySpec defines the spec of replication policy
+// Override policy should be used together with replication policy to select the deploy target components
+// ReplicationPolicySpec.Selector is the subset of selected components which will be replicated.
+type ReplicationPolicySpec struct {
```

Comment:
Add some comment for the Selector field that it will not drop components that not in the list. FYI, it might be a bit better if the description for each field is placed on the field, instead of the parent structure, but it is also acceptable here.